codebase/superdarn/src.bin/tk/tool/make_fit.1.19/make_fit.c

```diff
@@ -366,8 +371,12 @@ int main(int argc,char *argv[]) {
       /* If the allocation was successful, copy the parameters and */
       /* load the data into the FitACF structure.                  */
       if(fit_prms != NULL) {
-    	  Copy_Fitting_Prms(site,prm,raw,fit_prms);
-    	  Fitacf(fit_prms,fit);
+    	  if (prm->stid == 1)
```

Comment:
```suggestion
    	  if ((old_elev == 1) && (prm->stid == 1))
```

The `elev_goose` function should only be used for Goose Bay if the `-old_elev` option is specified and should not override the default v2 algorithm.

Reply:
Committing this suggestion now so that I can test it (and because it's correct :smile:)

Reply:
I switched it to match my code style as I don't use old_elev as it can be miss leading in context.